README.md

```diff
@@ -191,6 +191,26 @@ This command also has a dry run mode, which can be useful for initial setup and
 rdme changelogs:single path-to-markdown-file
 ```
 
+### Custom Pages
+
+#### Syncing a Folder of Custom Pages to ReadMe
```

Comment:
God, I wish these docs were pulled directly from the command files. `oclif` has something like this. Would be neat.

Reply:
Agreed 😭